rootfs/standard/var/www/mynode/templates/bitcoin_cli.html

```diff
@@ -47,9 +47,10 @@
             <form id="cmd_form" action="/bitcoin-cli/run">
                 <input class="ui-corner-all cli_command_input" id="cmd" name="cmd" type="text">
                 <input type="submit" class="ui-button ui-widget ui-corner-all cli_enter_button" value="Enter">
+		<small id="cli_small">Enter 'help' to see the list of possible commands.</small>
```

Comment:
This doesn't need to be a small tag if it's changed to class="cli_small"

Small could be changed to a <p> or <span>